Packs/Mattermost/Integrations/MattermostV2/MattermostV2.py

```diff
@@ -1617,7 +1688,129 @@ def send_notification(client: HTTPClient, **args):
         demisto.debug(f'MM: Found entitlement, saving message to context: {entitlement}')
         save_entitlement(entitlement, message_id, reply, expiry, default_response, to if to else channel_id)
     return CommandResults(
-        readable_output=f'Message sent to MatterMost successfully. Message ID is: {message_id}'
+        readable_output=f'Message sent to MatterMost successfully. Message ID is: {message_id}',
+        raw_response=raw_data
+    )
+
+
+def list_groups_command(client: HTTPClient, args: dict[str, Any]) -> CommandResults:
+    """ Lists groups """
+    page = arg_to_number(args.get('page', DEFAULT_PAGE_NUMBER))
+    page_size = arg_to_number(args.get('page_size', DEFAULT_PAGE_SIZE))
+    limit = arg_to_number(args.get('limit', ''))
+    q = args.get('group', '')
+    group_details: list[Any] = []
+    if limit:
+        page = DEFAULT_PAGE_NUMBER
+        page_size = limit
+
+    params = {'page': page, 'per_page': page_size}
+    if q:
+        params['q'] = q
+    group_details = client.list_groups_request(params)
+
+    hr = tableToMarkdown('Groups:', group_details, headers=['name', 'display_name', 'description', 'id'])
+    return CommandResults(
+        outputs_prefix='Mattermost.Groups',
+        outputs_key_field='id',
+        outputs=group_details,
+        raw_response=group_details,
+        readable_output=hr,
+    )
+
+
+def list_group_members_command(client: HTTPClient, args: dict[str, Any]) -> CommandResults:
+    """ List the members of a group """
+    group_id = args.get('group_id', '')
+    page = arg_to_number(args.get('page', DEFAULT_PAGE_NUMBER))
+    page_size = arg_to_number(args.get('page_size', DEFAULT_PAGE_SIZE))
+    limit = arg_to_number(args.get('limit', ''))
+    member_details = {}
+    if limit:
+        page = DEFAULT_PAGE_NUMBER
+        page_size = limit
+
+    params = {'page': page, 'per_page': page_size}
+    member_details = client.list_group_members_request(group_id, params)
+    member_details['id'] = group_id
+
+    hr = tableToMarkdown(f'Group {group_id} members ({len(member_details.get("members",[]))}/'
+                         f'{member_details.get("total_member_count", "Unknown")}):', member_details.get("members"),
+                         headers=['username', 'email', 'id'])
+    return CommandResults(
+        outputs_prefix='Mattermost.Groups',
+        outputs_key_field='id',
+        outputs=member_details,
+        readable_output=hr,
+        raw_response=member_details
+    )
+
+
+def add_group_member_command(client: HTTPClient, args: dict[str, Any]) -> CommandResults:
+    """ Adds member(s) to a group """
+    group_id = args.get('group_id', '')
+    user_ids = argToList(args.get('user_ids', ''))
+
+    data = {'user_ids': user_ids}
+    try:
+        response = client.add_group_member_request(group_id, data)
+    except Exception as ex:
+        if '"status_code": 500' in str(ex):
```

Comment:
Did we see this behavior to the add group user command? I noted to myself it was only in remove command.
can you retrieve the status_code from the Exception object instead of using if '"status_code": 500' in str(ex)?
In addition, lets write the error message to be more accurate- 
Got 500 from the server - this error could indicate that the user ID is invalid.